server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinatorConfig.java

```diff
@@ -51,6 +55,14 @@
   @Default("0")
   public abstract int getCoordinatorKillMaxSegments();
 
+  @Config("druid.coordinator.kill.audit.period")
+  @Default("P1D")
+  public abstract Duration getCoordinatorAuditKillPeriod();
+
+  @Config("druid.coordinator.kill.audit.durationToRetain")
+  @Default("PT-1s")
```

Comment:
These defaults don't seem to match the documented defaults above

Reply:
Fixed the docs. 
`druid.coordinator.kill.audit.period` is not required. And the default value is P1D
`druid.coordinator.kill.audit.durationToRetain` is required. The default value which is -1 which causes the precondition check to fail. User should set it to a positive value.